### PR TITLE
WIP: Added DevConf.CZ 2022 Talks and Presentations

### LINKED ISSUE #659: Add DevConf talk to presentations list
**Current State of the Meshery.io site**
List of presentations - https://meshery.io/talks-and-presentations

---
### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by adithyaakrishna**
Signed-off-by: Adithya Krishna <aadithya794@gmail.com>

**Description**

This PR fixes #659 

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — APPROVED** [review]

File: _data/talks.yml [hunk 1]
```diff
@@ -86,6 +88,26 @@
   presenter1_profile: https://layer5.io/community/members/lee-calcote
   new_window: true
 
+- title: Measuring Service Mesh Performance 101
+  video: https://www.youtube.com/watch?v=hzYy52A1VTw
+  embeded_video: https://www.youtube.com/embed/hzYy52A1VTw
+  presentation: TBA
```
Comment (leecalcote) [inline _data/talks.yml:94]: Until @navendu-pottekkat provides a link, this PR should not be merged. 

Either remove the entry for that talk or await his update.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: _data/talks.yml (hunk 1 above)
Comment (adithyaakrishna) [reply _data/talks.yml:94]: Yess, I will wait for an update from him 👀

File: _data/talks.yml [hunk 2]
```diff
@@ -86,6 +88,26 @@
...
+  presenter1_profile: https://layer5.io/community/members/navendu-pottekkat
+  new_window: true
+
+- title: Journey into the World of Service Meshes & Meshery
+  video: https://www.youtube.com/watch?v=WKijB2DkR8Y&t=2s
+  embeded_video: https://www.youtube.com/embed/WKijB2DkR8Y
+  presentation: https://docs.google.com/presentation/d/1XASGf0lP_3Oyzrix9tLgXcjydB3EBPi1/edit?usp=sharing&ouid=114101586100376807330&rtpof=true&sd=true
+  presenter1: Adithya Krishna V Sharma
+  presenter1_profile: https://layer5.io/community/members/adithya-krishna
+  presenter2: Nitish Karthik
```
Comment (sudo-NithishKarthik) [inline _data/talks.yml:105]:
```suggestion
  presenter2: Nithish Karthik
```

**Review by sudo-NithishKarthik — CHANGES_REQUESTED** [review]

**Review by sudo-NithishKarthik — APPROVED** [review]

File: _data/talks.yml (hunk 1 above)
Comment (leecalcote) [reply _data/talks.yml:94]: @adithyaakrishna no update thus far. I assume that there won't be. Will you refactor based on the lack of input?

**Comment by navendu-pottekkat** [comment]
@adithyaakrishna Sorry I missed this! Here is the [link to the presentation](https://docs.google.com/presentation/d/1pfIqorMVtaEDCIe999CF_QQ1KA5nKwoqqQVpltls45A/edit?usp=sharing). It is in the community drive.

**Comment by adithyaakrishna** [comment]
Thanks so much @navendu-pottekkat 🚀 
I have added your slide link as well :)

**Comment by adithyaakrishna** [comment]
@leecalcote @navendu-pottekkat @sudo-NithishKarthik Could you please review this PR as all the changes are addressed and Navendu's Presentation Link has also been added 💯

File: _data/talks.yml (hunk 1 above)
Comment (adithyaakrishna) [reply _data/talks.yml]: Got an Update from him (had missed his notification) and added link to the slide as well 👀

**Review by warunicorn19 — APPROVED** [review]
Looking good, Thank you @adithyaakrishna

**Review by sudo-NithishKarthik — APPROVED** [review]

**Comment by adithyaakrishna** [comment]
Lessssgooooo 🎉 
Thanks much @warunicorn19  :)

**Pull request merged**